rust/otap-dataflow/benchmarks/README.md

```diff
@@ -0,0 +1,81 @@
+# 📈 Benchmarks
```

Comment:
Need to remove all emojis/non-ascii characters, else the sanitycheck CI action will fail!